src/handlers/prioritize.rs

```diff
@@ -39,18 +39,23 @@ impl Handler for PrioritizeHandler {
                         // We need to take the exact same action in this case.
                         return Ok(Some(Prioritize::Start));
                     } else {
-                        match glob::Pattern::new(&config.priority_labels) {
-                            Ok(glob) => {
-                                let issue_labels = event.issue().unwrap().labels();
-                                let label_name = &e.label.as_ref().expect("label").name;
+                        let label_name = &e.label.as_ref().expect("label").name;
 
-                                if issue_labels.iter().all(|l| !glob.matches(&l.name))
-                                    && config.prioritize_on.iter().any(|l| l == label_name)
-                                {
-                                    return Ok(Some(Prioritize::Label));
+                        if config.prioritize_on.iter().any(|l| l == label_name)
+                            && config.exclude_labels.iter().all(|exclude_label| {
+                                match glob::Pattern::new(exclude_label) {
+                                    Ok(glob) => {
+                                        let issue_labels = event.issue().unwrap().labels();
```

Comment:
Let's extract to avoid `.unwrap()` in the loop?

Reply:
This is out of the loop in the new code.